kibana/kibana.repo

```diff
@@ -0,0 +1,6 @@
+[kibana-4.5]
```

Comment:
good catch, i had meant to update the kibana install to use the rpm again but lost track of that